ui/desktop/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## v1.7.0 [CLI 0.14.0] (2023.10.09)
+
+### New
+- Upgrade CLI to v0.14.0 ([PR]())
+- Update Connection workflow ([PR](https://github.com/hashicorp/boundary-ui/pull/1919))
```

Comment:
None blocking comment: I guess the refactor of sessions listing is part of the connection workflow update? To me sounds right, but wanted to double check 😉

Reply:
Correct. I felt it was more concise to just link to the llb branch being merged into main to show all of the work. I felt if I started to pull out specific changes, I would likely keep pulling out each change.